core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -35,6 +35,7 @@
 import static com.google.common.base.Preconditions.checkNotNull;
 import static com.google.common.base.Preconditions.checkState;
 
+import com.google.census.CensusContextFactory;
```

Comment:
Well, I have a concern mostly about the dependencies that census might bring in at this point.  I think if there was a census-api package that had zero dependencies and is all gRPC had to take a dependency on I'd be more comfortable.  I also worry about small things like guava being at 2 different versions between gRPC and census as well if they can't be resolved, though that's probably no concern given google ownership of both.